ls1intum/Artemis
# [Bugfix] - Additional Feedback consistency

Fixes #1708




### Checklist

### Motivation and Context


We handled additional feedback (or at is was called before, unreferenced feedback) differently for every exercise type. This should be consistent.

### Description

- Added a additonal-feedback component which shows the additional feedback done by the assessor and this component is used by every exercise type

### Steps for Testing


1. Log in to Artemis
2. For text-/modeling-/file-upload- and programming exercises do:
2.1. Create an exercise and participate
2.2. Assess the submission (once with additional feedback (one or multiple) and once without additional feedback)
2.3. With the student, go to the submission, below the submission, the additional feedback should be shown in the same way for every exercise or should be not shown if no additional feedback provided

Aspects to also review:
-  File upload exercises have some wording adaptions, see screenshot
- For modeling exercises, the additional feedback should not be shown in the feedback list next to the editor (At the beginning)

### Screenshots


Text Exercise:
![Screenshot 2021-03-03 at 14 06 49](https://user-images.githubusercontent.com/44401560/109810541-f15f7380-7c29-11eb-8749-c175d61715db.png)

Modeling Exercise:
![Screenshot 2021-03-03 at 14 06 34](https://user-images.githubusercontent.com/44401560/109810513-e7d60b80-7c29-11eb-95c7-2b92bc83b511.png)

File-Upload Exercise:
![Screenshot 2021-03-03 at 14 06 56](https://user-images.githubusercontent.com/44401560/109810563-f9b7ae80-7c29-11eb-9a4e-8816b1973c74.png)

Programming Exercise:
![Screenshot 2021-03-03 at 22 07 27](https://user-images.githubusercontent.com/44401560/109872811-38bb2380-7c6d-11eb-80e1-b91403c4d164.png)
